ports/esp32s2/boards/lilygo_ttgo_t8_s2_st7789/pins.c

```diff
@@ -54,5 +55,8 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
 
     // Peripheral Power control
     { MP_ROM_QSTR(MP_QSTR_PE_POWER), MP_ROM_PTR(&pin_GPIO14) },
+
+    // Battery Sense
+    { MP_ROM_QSTR(MP_QSTR_VBAT_SENSE), MP_ROM_PTR(&pin_GPIO9) },
```

Comment:
I think `VOLTAGE_MONITOR` or `BATTERY` are the standard names for this. How about renaming this `BATTERY`?

Reply:
Sure, I copied the name from another board. But BATTERY it is, or will be soon. Thanks for the quick review. I'll make the change ASAP.